editor/src/components/inspector/sections/image-section/image-source-control.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import { imagePathURL } from '../../../../common/server'
+import { treeToContents } from '../../../../components/assets'
+import { isImageFile } from '../../../../core/model/project-file-utils'
+import { ProjectContents } from '../../../../core/shared/project-file-types'
+import { useEditorState } from '../../../editor/store/store-hook'
+import { useInspectorElementInfo } from '../../common/property-path-hooks'
+import { OptionChainControl } from '../../controls/option-chain-control'
+import { SelectControl, SelectOption } from '../../controls/select-control'
+import { StringControl } from '../../controls/string-control'
+
+enum ImageSrcType {
+  URL = 1,
+  LOCAL,
+}
+
+const getProjectImageFileNames = (projectContents: ProjectContents): string[] => {
+  return Object.keys(projectContents).filter((key) => isImageFile(projectContents[key]))
+}
+
+const srcTypeFromSrcValue = (value: string, filenames: string[]): ImageSrcType => {
```

Comment:
Aren't these just regular functions?

Reply:
yes, would you prefer to have them as `function srcTypeFromSrcValue`? can I ask you why? I don't remember the reason, but in the past I've been told it's considered "a best practice" to have them as constants. Any enlightenment for this n00b here would be awesome :)